.github/workflows/test-package.yml

```diff
@@ -56,9 +56,9 @@ jobs:
       - id: install
         name: Install dependencies
         run: dart pub get
-      - name: Run VM tests
-        run: dart test --platform vm --test-randomize-ordering-seed=random
-        if: always() && steps.install.outcome == 'success'
-      - name: Run Chrome tests
-        run: dart test --platform chrome --test-randomize-ordering-seed=random
+      - name: Run tests
+        run: dart test -p chrome,vm --test-randomize-ordering-seed=random
         if: always() && steps.install.outcome == 'success'
+      - name: Run Legacy tests
+        run: dart test -p chrome,vm --test-randomize-ordering-seed=random legacy_test
+        if: always() && matrix.sdk == '2.14.0' && steps.install.outcome == 'success'
```

Comment:
Perhaps also add a comment, or or around line 50, that when updating the 2.14 value there to also update here?

Reply:
Done.